web-app-aspnet/appsettings.json

```
@@ -13,11 +13,13 @@
   "CallbackPath": "/signin-oidc"
 },
   "DownstreamApi": {

```

```suggestion
  "DownstreamApis": {
```

Does this also need to be updated?